rel-sign.sh

```diff
@@ -315,11 +315,22 @@ sign() {
 			ecdo gpg --armor "${PPARAMS[@]+${PPARAMS[@]}}" --output "v${VERSION}.${ext}.asc" --detach-sign "${TMPDIR}/v${VERSION}.${ext}"
 		done
 	fi
+	# Calculate hashes as well
+	pushd "${TMPDIR}" > /dev/null
+	ecdo sha256sum "v${VERSION}.zip" "v${VERSION}.tar.gz" > "v${VERSION}.sha256"
```

Comment:
I would be careful about using `ecdo` with redirections...

The result file would contain the echoed command at the top!

Better split up the `ecdo` call into two:

```
 	echo sha256sum "v${VERSION}.zip" "v${VERSION}.tar.gz" ">" "v${VERSION}.sha256"
 	sha256sum "v${VERSION}.zip" "v${VERSION}.tar.gz" > "v${VERSION}.sha256"
```

Reply:
@ribose-jeffreylau Thanks! I guess first one should be `ecdo echo sha256sum ...` or do I miss something?

Reply:
In the code block I gave, I put `"` quotes around `>` so it (`">"`) can be treated as an argument to `echo`, so the `>` character can be printed out in the console.

If we don't quote `>`, then the line `sha256sum v0.15.2.zip v0.15.2.tar.gz` would be at the top of the file `v0.15.2.sha256` due to `>` being interpreted by bash as a redirection, which is not what we want.

The second line is just what we want the script to actually perform.

Hope it makes sense! :p

Reply:
@ribose-jeffreylau Got it, thanks. Actually we should not use `echo` but `infop`, so output would be printed only if verbose/debug mode is enabled?

Reply:
Yes, `infop` would work, too!

or use `info` which is shorter and appends a new line for you :D

Reply:
@ribose-jeffreylau Thanks! Fixed and force-pushed! Do you have any other comments on this?

Reply:
Thanks @ni4!

Looks good.

Reply:
@ni4 Can merge when ready :)

Reply:
@ribose-jeffreylau Thanks, so, merging now!